_data/publications.yml

```diff
@@ -24,8 +24,8 @@
   title: "Laue-DIALS: open-source software for polychromatic X-ray diffraction data"
   first_author: "Hewitt"
   authors: "**, Rick A.** and Kevin M. Dalton, Derek Mendez, Harrison K. Wang, Margaret A. Klureza, Dennis E. Brookner, Jack B. Greisman, David McDonagh, Vukica Šrajer, Nicholas K. Sauter, Aaron S. Brewster, Doeke R. Hekstra"
-  url: https://www.biorxiv.org/content/10.1101/2024.07.23.604358v1
-  info: bioRxiv (preprint)
+  url: https://pubs.aip.org/aca/sdy/article/11/5/054701/3315187/Laue-DIALS-Open-source-software-for-polychromatic
```

Comment:
this URL should use the DOI, [https://doi.org/10.1063/4.0000265](https://doi.org/10.1063/4.0000265)